.circleci/config.yml

```diff
@@ -6,13 +6,21 @@ maven_cache_key: &maven_cache_key maven-{{ checksum "miniapp/src/test/AndroidMan
 jobs:
   build:
     docker:
-      - image: circleci/android:api-28
+      - image: circleci/android@sha256:45eb47efaf498cdbf01b16e6a294ef20a4979f6acd6aaf75913707190806bb4c
     working_directory: ~/code
     environment:
       JVM_OPTS: "-Xmx3200m"
     steps:
       - checkout
-      - run: git submodule update --init
+      - run:
+          name: Update submodule
+          command: |
+            set -e
+            git submodule update --init
+            cp miniapp/src/main/assets/js-miniapp/js-miniapp-bridge/export/android/bridge.js miniapp/src/main/assets
+            rm -r miniapp/src/main/assets/js-miniapp
+            (cd miniapp/src/main/assets && mkdir -p js-miniapp/js-miniapp-bridge/export/android)
+            mv miniapp/src/main/assets/bridge.js miniapp/src/main/assets/js-miniapp/js-miniapp-bridge/export/android/
```

Comment:
Too bad this takes so many lines of code 😣 

Maybe add a `# TODO:` comment about finding a workaround to use `sparse-checkout` or updating this after the new version of Git is available in for Debian?

Reply:
Yes, actually, I found this https://circleci.com/docs/2.0/executor-intro/#machine
It looks like we can point the docker to use Ubuntu instead of Debian.

We need a ticket to try this in CI.